api/envoy/config/filter/udp/dns_filter/v2alpha/dns_filter.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package envoy.config.filter.udp.dns_filter.v2alpha;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.udp.dns_filter.v2alpha";
+option java_outer_classname = "DnsFilterProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.udp.dns_filter.v3";
+option (udpa.annotations.file_status).work_in_progress = true;
+
+// Configuration for the DNS filter.
+message DnsFilterConfig {
+  // Specify the method of determining the address of a given endpoint
+  enum EndpointType { STATIC = 0; }
+
+  // This message type is extensible and can contain a list of addresses
+  // or dictate some other method for resolving the addresses for an
+  // endpoint
+  message DnsEndpoint {
+    EndpointType type = 1;
+
+    // A list of IP addresses Envoy returns to queries for the given domain
+    repeated string address = 2;
```

Comment:
These should be `core.Address` messages I think.

Reply:
I looked at those, but I felt strings were better suited here.  It seems that `core.Address` needs more fields defined that aren't required here.

Reply:
Generally we prefer to have structured representation of addresses; if you plan on doing any kind of address parsing or transform prior to returning this in a DNS query, you probably want this as Envoy's existing utilities are geared to taking `Address`. If it's 100% opaque forever, I guess `string` is fine here.

Reply:
I will stick to `string` for now but, I will try move to `Address` and see how that works.  I have to do address parsing and conversions so `Address` definitely makes sense here.